src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs

```diff
@@ -35,6 +35,11 @@ protected override string AddFromMagnetLink(RemoteEpisode remoteEpisode, string
         {
             var actualHash = _proxy.AddTorrentFromMagnet(magnetLink, Settings);
 
+            if (actualHash.IsNullOrWhiteSpace())
+            {
+                throw new DownloadClientException("Deluge failed to add magnet " + magenetLink);
```

Comment:
magenetLink (typo) 😃

Reply:
That's what I get for making this change directly in github's ui...